spring-boot/proxy/src/main/java/eu/coatrack/proxy/security/ApiKeyAuthenticator.java

```diff
@@ -129,8 +129,8 @@ private ApiKey getApiKeyEntityByApiKeyValue(String apiKeyValue) {
         try {
             apiKey = apiKeyFetcher.requestApiKeyFromAdmin(apiKeyValue);
         } catch (ApiKeyFetchingFailedException e) {
-            log.debug("Trying to verify consumers API key with the value {}, the connection to admin failed. " +
-                    "Therefore checking the local API key list as fallback solution.", apiKeyValue);
+            log.info("An error occurred for api key {}. " +
```

Comment:
Why remove the information that "fetching from admin" failed? That is actually the reason to check locally.